ports/wordswithfreds/Words With Freds.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+source $controlfolder/control.txt
+source $controlfolder/device_info.txt
+export PORT_32BIT="Y"
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+get_controls
+
+$ESUDO chmod 666 /dev/tty0
+
+GAMEDIR=/$directory/ports/wordswithfreds
+
+exec > >(tee "$GAMEDIR/log.txt") 2>&1
```

Comment:
Try to use the new logging which overwrites itself 

```> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1```

Reply:
Okies, Updated, please review